### PR TITLE
Add unit to livetime

### PR DESCRIPTION
**Pull request by pdeiml**
make.py checks 
runs without errors, hence the changes should be fine.
Do you know a better commit description?

### CODE REVIEW

**Comment by cdeil** [comment]
Can you also try `./make.py all` or `./make.py catalog`?

I think the line here:
https://github.com/gammapy/gamma-cat/blob/ecb4648cf94a6cb7f31cb6070763a6437d1f9d96/gammacat/catalog.py#L248
has to be changed to 
```
data['livetime'] = Quantity(dsi['data']['livetime']).to('hour').value
```
so that it works, no?

After this change, the output catalog shouldn't change with this PR.
This is something you can check via `git diff docs/data/gammacat.yaml` or `git diff docs/data/gammacat.ecsv`, to see what changed in the generated output catalog.

**Comment by cdeil** [comment]
To check if unit handling for this input really works, you could also change one of the livetimes e.g. from "1 hour" to "60 min" and in the output files it should be "1 hour" in both cases.

**Comment by pdeiml** [comment]
You were right. I changed line 248 in 
gamma-cat/gammacat/catalog.py. 
Now make.py all does work.

I checked the unit conversion by changing the lifetime of 41.25h in 2013/2013ApJ...764...38A/tev-000001.yaml
to 2475min and in both cases in gammacat.fits.gz the corresponding entry is 41.25.

So this works and it is ready to merge. For the other conversion, I think, a new PR is better than to put everything in this PR.

By the way: How did you do this nice link to a specific line in a file in you upper comment?

**Comment by pdeiml** [comment]
The last commit was necessary to fix some merge conflicts :-)

**Comment by cdeil** [comment]
This is not OK, I don't want to merge this:
https://github.com/gammapy/gamma-cat/pull/141/commits
It will lead to very complex version history and is complicated / time consuming to review.

I think we've discussed it before, but please: don't merge master into your feature branch; instead rebase your feature branch on master.

**Comment by cdeil** [comment]
> By the way: How did you do this nice link to a specific line in a file in you upper comment?

I just linked to a line of code, and then Github inserted that for me that way:
https://github.com/blog/2415-introducing-embedded-code-snippets

**Comment by pdeiml** [comment]
Ok, nice this thing with the linked lines :-)

Now this PR is ready to merge.

**Comment by cdeil** [comment]
> Now this PR is ready to merge.

No. This PR shows commits from Gernot and me, because you merged master into this branch. Please see my comment on that matter above and fix.

**Comment by pdeiml** [comment]
But now, it is ready to merge

File: input/data/2013/2013ApJ...764...38A/tev-000001.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ reference_id: 2013ApJ...764...38A
 telescope: veritas
 
 data:
-  livetime: 41.25
+  livetime: 2475min
```
Comment (cdeil) [inline input/data/2013/2013ApJ...764...38A/tev-000001.yaml]: Please go back to hours here. Trying out minutes was just for testing.

**Comment by pdeiml** [comment]
ah sry, that was from the unit testing. 
Fixed it

**Comment by pdeiml** [comment]
Think it is ready to merge.

**Comment by cdeil** [comment]
Looks good. Thanks!

**Pull request merged**